pkg/yang/entry_test.go

```diff
@@ -1658,6 +1658,310 @@ func TestActionRPC(t *testing.T) {
 	}
 }
 
+var testIfFeatureModules = []struct {
+	name string
+	in   string
+}{
+	{
+		name: "if-feature.yang",
+		in: `module if-feature {
+  namespace "urn:if-feature";
+  prefix "feat";
+
+  container cont {
+    if-feature ft;
```

Comment:
Would you mind adding/changing these feature names to match their usage, so that the same feature label/name is used only for one test assertion?

e.g., for this case: `if-feature ft-container;`